modules/openapi-generator/src/main/resources/JavaJaxRS/spec/pojo.mustache

```diff
@@ -1,52 +1,38 @@
-{{#useSwaggerAnnotations}}
-import io.swagger.annotations.*;
-{{/useSwaggerAnnotations}}
+{{#useSwaggerAnnotations}}import io.swagger.annotations.*;{{/useSwaggerAnnotations}}
```

Comment:
Please revert this change to avoid a blank line when `useSwaggerAnnotations` is false.